src/components/SingletonEditor.tsx

```diff
@@ -102,6 +104,48 @@ export const useSingletonEditor = (
     [editorState, ...deps]
   );
 
+  const exportHandler = async () => {
+    if (!editorState.isLoaded) {
+      console.error("The editor is loading, can't save yet!");
+      return;
+    }
+    if (!onExport) {
+      console.error("The onExport callback must be configured");
+      return;
+    }
+
+    setIsExporting(true);
```

Comment:
It should be set to `false` again in case of an error, right?

Reply:
I am talking about [line #131](https://github.com/FromDoppler/doppler-editors-webapp/pull/412/files#diff-5886b9902d1e2302004c7c6185c63c9950e6aa9cb7a3739434720fc70be3eee5R131) `if (!htmlExport.design) {`

Reply:
This was move to campaing implementation.